applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -103,6 +106,20 @@ config DESKTOP_BLE_DONGLE_PEER_SELECTOR_POS
 	range 0 255
 	default 0
 
+endif
+
+if !DESKTOP_BLE_DONGLE_PEER_ENABLE
+
+config DESKTOP_BLE_DONGLE_PEER_SELECTOR_ID
+	int
+	default 0
+
+config DESKTOP_BLE_DONGLE_PEER_SELECTOR_POS
+	int
+	default 0
```

Comment:
Would be nice with some help text here, so that users are not forced to go to the docs for additional info.

Reply:
Ok, I will add help here.